core/integration/steps/time_is_updated.go

```diff
@@ -13,11 +13,14 @@
 package steps
 
 import (
+	"context"
+
 	"code.vegaprotocol.io/vega/core/integration/stubs"
 )
 
-func TimeIsUpdatedTo(timeService *stubs.TimeStub, newTime string) {
+func TimeIsUpdatedTo(exec Execution, timeService *stubs.TimeStub, newTime string) {
 	t, err := Time(newTime)
 	panicW("date", err)
 	timeService.SetTime(t)
+	exec.BlockEnd(context.Background())
```

Comment:
I don't think that is correct. it may be harmless but if someone defined average block time to be 2 seconds and they move the time by 10 seconds, they'd expect blockEnd to be called 5 times. Similarly, if they set average block time to 5 seconds and they move the time by 3 seconds it shouldn't be called at all.

Reply:
The `TimeIsUpdatedTo` is a step that predates the steps like `Then the network moves ahead "N" blocks`. These steps don't use block duration, and just move time forwards to a specific point in a single jump. They're still there because in older scenario's (e.g. price monitoring) they are still used, but gradually scenario's are being reworked to do away with this all together, and instead use blocks to progress time. For that reason, I'd say this is a non-issue here